### DISCUSSION TITLE
mongodb plugins support has only params?  has no document name?

### DISCUSSION THREAD

**Discussion by lanxiudh** [apache/skywalking#7715] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/15043144/133364082-70ec4496-1384-489a-ad68-12bcde2cbc03.png)

How can i show the document name?

**Comment by wu-sheng** [comment, marked as answer]
Currently, there is no parameter to control that. Plugins are contributed by the whole community(hundreds of people). It is common some are there or not there. If you want to enhance some, feel free to submit a pull request by following plugin dev doc requirements.
